core/domain/question_services.py

```diff
@@ -571,7 +574,10 @@ def apply_change_list(question_id, change_list):
             '%s %s %s %s' % (
                 e.__class__.__name__, e, question_id, change_list)
         )
-        raise
+        # This code is needed in order to reraise the error properly with
+        # the stacktrace. See https://stackoverflow.com/a/18188660/3688189.
+        exec_info = sys.exc_info()
+        six.reraise(exec_info[0], exec_info[1], tb=exec_info[2])
```

Comment:
After seeing the third occurrence, I definitely think abstracting this would be nice since you can also keep the context of why this is needed in one location.

Reply:
Done.